README.md

```diff
@@ -21,6 +21,14 @@ Polaris helps keep your cluster healthy. It runs a variety of checks to ensure t
 
 **Want to learn more?** ReactiveOps holds [office hours on Zoom](https://zoom.us/j/951540265) the first Friday of every month, at 12pm Eastern. You can also reach out via email at `opensource@reactiveops.com`
 
+## Quickstart
```

Comment:
There are documentation changes not noted in the PR change set (non-blocking comment).

Reply:
good catch, fixed!